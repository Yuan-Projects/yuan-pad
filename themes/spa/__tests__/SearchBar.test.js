import renderer from 'react-test-renderer';
import SearchBar from '../src/front/searchBar';

it('SearchBar renders correctly', () => {
  const props = {
    onSubmit: (s) => {},
  };
  const component = renderer.create(<SearchBar {...props} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

  // const component2 = renderer.create(<SearchBar {...props} />);
  // let tree2 = component2.toJSON();
  // expect(tree2).toMatchSnapshot();
});
