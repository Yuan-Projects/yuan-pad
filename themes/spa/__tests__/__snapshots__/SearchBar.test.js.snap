// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar renders correctly 1`] = `
<div
  className="searchbar"
>
  <form
    className="form-inline"
    onSubmit={[Function]}
  >
    <div
      className="form-group"
    >
      <label
        className="sr-only"
        htmlFor="inputSearch"
      >
        Search
      </label>
      <input
        className="form-control"
        id="inputSearch"
        onChange={[Function]}
        placeholder="Search"
        size={10}
        type="text"
        value=""
      />
    </div>
    <button
      aria-label="Submit"
      className="btn btn-default btn-sm"
      onClick={[Function]}
      type="button"
    >
      <span
        aria-hidden="true"
        className="glyphicon glyphicon-search"
      />
    </button>
  </form>
</div>
`;
